src/styles/Detailspage.css

```diff
@@ -0,0 +1,121 @@
+.detailspage {
+  font-family: Arial, sans-serif;
+}
+
+.parent {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-content: center;
+  width: 100%;
+  height: 100vh;
+}
+
+.items {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(54, 37, 114, 0.518);
+  text-align: center;
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up_2: please stick to the Behance design but still your design is not matching. 
- [ ] As mentioned by the previous reviewer :point_up_2: we need to choose two major colors and stick to those colors. currently, there are (white, black, red, and light brown) colors on your design, beside that the background white is still visible, this is a bit odd compared to the original design but still you using (white, black, red, and light brown) colors so kindly choose two major colors and stick to those colors.
- [ ] As mentioned by the previous reviewer :point_up_2: As you can see from the original design there is no margin between elements. Please try to remove the white background and stick to the original design but still there are some margins and white background so kindly remove background color and also remove margins.

Check below :point_down:  the comparison between your design and Behance's design :slightly_smiling_face:

**Your Design**
![Screenshot from 2023-07-01 15-48-30](https://github.com/waelelsafty07/Metrics-webapp/assets/45226848/bc436618-b7bb-4776-a2db-1f6f31ae8012)

**Behance Design** 
![image](https://github.com/waelelsafty07/Metrics-webapp/assets/45226848/b1cf342a-379d-479c-ad98-aa547b52a121)

Reply:
@aamirkhan2478  bro are what r u say?

Reply:
@waelelsafty07 let me call you in zoom okay bro